wallet/src/Pos/Util/Mnemonic.hs

```diff
@@ -294,12 +286,20 @@ instance Buildable (SecureLog (Mnemonic mw)) where
     build _ =
         "<mnemonic>"
 
-instance Buildable MnemonicErr where
+instance Buildable Basement.String where
+    build = bprint . show . fromUtf8String
```

Comment:
This feels wrong. There's no need for this orphan instance here. Rather go for an explicit use of `fromUtf8String`.